src/timelines.js

```diff
@@ -483,6 +484,10 @@ export default Kapsule({
         .direction('s')
         .offset([5, 0])
         .html(d => {
+          if (state.segmentTooltipHtml) {
+            return state.segmentTooltipHtml(state, d);
```

Comment:
`state` is internal to the library. It should not be passed as part of this method, only the segment data.

Reply:
makes sense, i'll adjust